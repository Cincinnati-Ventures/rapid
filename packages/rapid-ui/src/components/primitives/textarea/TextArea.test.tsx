import React from 'react';
import { render } from '@testing-library/react';
import { expect, it, describe } from 'vitest';
import TextArea from './Textarea';

describe('TextArea', () => {
	it('should render default correctly', () => {
		const { container } = render(<TextArea />);
		expect(container).toMatchSnapshot();
	});
});
