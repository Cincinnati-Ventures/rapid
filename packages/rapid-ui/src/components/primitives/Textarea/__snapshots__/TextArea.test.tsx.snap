// Vitest Snapshot v1

exports[`TextArea > should render default correctly 1`] = `
<div>
  <textarea
    class="rapid-textarea"
    placeholder="Rapid Textarea"
  />
</div>
`;
